scripts/net/run-sample-tests.sh

```diff
@@ -394,6 +404,17 @@ do
 	    fi
 	    configuration=stop_only
 	    ;;
+
+	--overlay)
+	    shift
+	    if [ -n "$zephyr_overlay" ]
```

Comment:
Can't we just say `zephyr_overlay="$zephyr_overlay $1"` here? Is there some shell that does not accept this?

Reply:
We get an extra space into that variable in such a case, but that should mean harm.